core/src/main/resources/AppConstants.properties

```diff
@@ -180,6 +180,10 @@ xml.namespaceAware.default=false
 xslt.streaming.default=true
 xslt.auto.reload=false
 xslt.bufsize=4096
+# ForEachChildElement pipe usually works with large files and uses streaming.
+# Currently streaming is only supported with xslt version 1.
+# Therefore this property will not change the xslt version in ForEachChildElementPipe.
+# If desired, xsltVersion can be set for the individual pipe.
```

Comment:
This is a confusing comment if you put is this way. Please first explain/document the property as it is, and then note the exception